data/users.js

```diff
@@ -42,6 +43,7 @@ let exportedMethods = {
     // Add libraryId to user's favorited libraries
     const userCollection = await users();
 
+    const user = await userCollection.findOne({ _id: new ObjectId(userId) });
     const user = await userCollection.findOne({ _id: new ObjectId(userId) });
```

Comment:
Hey the server is crashing because of this since u are redeclaring a const
please check this